components/date-picker/interface.tsx

```diff
@@ -23,21 +23,22 @@ export interface PickerProps {
   onOpenChange?: (status: boolean) => void;
   disabledDate?: (current: moment.Moment | undefined) => boolean;
   dateRender?: (current: moment.Moment, today: moment.Moment) => React.ReactNode;
+  autoFocus?: boolean;
 }
 
 export interface SinglePickerProps {
   value?: moment.Moment;
   defaultValue?: moment.Moment;
   defaultPickerValue?: moment.Moment;
-  renderExtraFooter?: (mode: 'date' | 'month' | 'year' | 'decade') => React.ReactNode;
+  placeholder?: string;
+  renderExtraFooter?: (mode: DatePickerMode) => React.ReactNode;
```

Comment:
`DatePickerMode | 'decade'`， `decade` is not export as mode.

Reply:
I don't understand this. Do you mean that `DatePickerMode` (which is exported) should not include `'decade'`?

But `'decade'` is also a valid value for the `mode` prop ([CodeSandbox example](https://codesandbox.io/s/0q5kmwr7kl)), so it should be included in `DatePickerMode`...

Reply:
It's not in the document since we currently not export to user:
<img width="689" alt="屏幕快照 2019-05-03 下午2 14 31" src="https://user-images.githubusercontent.com/5378891/57122076-e4636600-6dad-11e9-8d47-ae34a3dcdf0f.png">

Reply:
Ok, then the docs are incomplete and should be fixed:

There is definitely a "decade" mode the user can use a value for the mode prop, and `renderExtraFooter()` then gets "decade" as first argument (parameter "mode). See the CodeSandbox linked in my previous comment (<https://codesandbox.io/s/0q5kmwr7kl>), where mode prop is pinned to "decade".

You can also take a look at the more complete CodeSandbox <https://codesandbox.io/s/388xplj646>

1. activate first date picker
2. click on year in header
3. click on decade in header
4. take a look at the footer

Reply:
I fixed the docs, but accidentally pushed too much into this PR branch😠

Sorry, will fix it in the next minutes.

Reply:
Fixed